mycroft/res/text/en-us/date_time_test.json

```diff
@@ -0,0 +1,34 @@
+{
+  "test_nice_year": {
+      "1": {"datetime_param": "2017, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "two thousand and seventeen"},
+      "2": {"datetime_param": "1984, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "nineteen hundred and eighty four"},
+      "3": {"datetime_param": "1906, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "nineteen hundred and six"},
+      "4": {"datetime_param": "1802, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "eighteen hundred and two" },
+      "5": {"datetime_param": "806, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "eight hundred and six" },
+      "6": {"datetime_param": "1800, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "eighteen hundred" },
+      "7": {"datetime_param": "1000, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "one thousand" },
+      "8": {"datetime_param": "2000, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "two thousand" },
+      "9": {"datetime_param": "99, 1, 31, 13, 22, 3", "bc": "True", "assertEqual": "ninety nine b.c." },
+      "10": {"datetime_param": "5, 1, 31, 13, 22, 3", "bc": "True", "assertEqual": "five b.c." },
+      "11": {"datetime_param": "3120, 1, 31, 13, 22, 3", "bc": "True", "assertEqual": "three thousand one hundred and twenty b.c." },
+      "12": {"datetime_param": "3241, 1, 31, 13, 22, 3", "bc": "True", "assertEqual": "three thousand two hundred and forty one b.c." },
+      "13": {"datetime_param": "1001, 1, 31, 13, 22, 3", "bc": "False", "assertEqual": "one thousand and one" },
+      "14": {"datetime_param": "2100, 1, 31, 13, 22, 3", "bc": "False", "assertEqual": "two thousand one hundred" },
+      "15": {"datetime_param": "5200, 1, 31, 13, 22, 3", "bc": "False", "assertEqual": "five thousand two hundred" }
+  },
+  "test_nice_date": {
+      "1": {"datetime_param": "2017, 1, 31, 0, 2, 3", "now": "None", "assertEqual": "tuesday, january the thirty-first, two thousand and seventeen"},
```

Comment:
This is OK, but I think I'd leave out "the" most of the time for dates.  It can sound alright in some situations, but in general the output is more normal as "The concert is January first" or "The concert is January twenty third".